README.md

```diff
@@ -1 +1,77 @@
-# covid-app
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+Covid Tracker Application
+
+![screenshot](./demo.png)
+![screenshot](./demo1.png)
+![screenshot](./demo2.png)
+
+# Project Description
+
+> The Covid-stats-app allows users to have access to the latest covid data in Us. They can also filter by region name, You can also get more informations on that specific region by clicking on it.
+## Built With
+
+- Major languages
+    - JavaScript
+    - HTML & CSS
+- Frameworks
+    - React
+- State Management
+    - Redux
+- Text editor
+    - VSCode
+- Api
+    - - [Narrativa API](https://covid19tracking.narrativa.com/index_en.html)
+
+## Demo
+- [live demo] (https://deploy-preview-2--infallible-mccarthy-668a73.netlify.app/)
```

Comment:
- [ ] [OPTIONAL] You did great deploying your app but to style it in your README file I think you should remove the space after the `]` and it should display the `live demo` only text which will take us to the project demo instead of displaying the whole url